Code/Tools/SerializeContextTools/Dumper.cpp

```diff
@@ -151,9 +250,26 @@ namespace AZ::SerializeContextTools
 
     bool Dumper::DumpTypes(Application& application)
     {
-        // outputStream defaults to a stdout stream
+        // outputStream defaults to writing to stdout
+        auto WriteBypassStdoutCapturer = [&application](AZStd::span<AZStd::byte const> outputBytes)
+        {
+            // If the application is currently capturing stdout, use stdout capturer to write
+            // directly to the file descriptor of stdout
+            if (AZ::IO::FileDescriptorCapturer* stdoutCapturer = application.GetStdoutCapturer();
+                stdoutCapturer != nullptr)
+            {
+                return stdoutCapturer->WriteBypassingCapture(outputBytes.data(), aznumeric_caster(outputBytes.size()));
+            }
+            else
+            {
+                constexpr int StdoutDescriptor = 1;
+                return AZ::IO::PosixInternal::Write(StdoutDescriptor, outputBytes.data(), aznumeric_caster(outputBytes.size()));
+            }
+
+        };
```

Comment:
if this lambda function gets used more in this file, might consider pulling it out into a re-usable function